try:
    print(name)
except NameError as e:
    print(e)
